src/components/Kv/KvLightbox.vue

```diff
@@ -147,7 +147,6 @@ export default {
 	bottom: 0;
 	padding: rem-calc(1);
 	overflow-y: auto;
-	-webkit-overflow-scrolling: touch;
```

Comment:
Removing these since Safari 13[ removed the need for them](https://developer.apple.com/documentation/safari-release-notes/safari-13-release-notes).

Reply:
Sept 2019 seems like a decent grace period.